src/pages/GetStarted/index.tsx

```diff
@@ -169,6 +170,7 @@ export class GetStarted extends React.PureComponent<Props, State> {
     return (
       <React.Fragment>
         <Head pageName="Get Started" />
+        <WebSocketBannerAlert />
```

Comment:
@JPinkney you don't need to add the `<WebSocketBannerAlert />` to each page. 

Add it once above this line:
https://github.com/che-incubator/che-dashboard-next/blob/7f0263f70e5b8a35159d5aa334ae0360a06025c7/src/Layout/index.tsx#L152